CMakeLists.txt

```diff
@@ -1021,7 +1021,6 @@ if(WITH_TESTS)
         utilities/option_change_migration/option_change_migration_test.cc
         utilities/options/options_util_test.cc
         utilities/persistent_cache/hash_table_test.cc
-        utilities/persistent_cache/persistent_cache_test.cc
```

Comment:
This is a little bit too wide. It not only removes the test from Windows, but cmake for all platforms too. Can we append it back if it is not for Windows?

Reply:
Yeah. Let me conditionally compile out the persistent_cache_test.cc code on Windows, then revert this part of the change. It's kind of unfortunate, though, because ideally persistent cache wouldn't fail on `GetUniqueId()` returning zero.